cluster-autoscaler/simulator/cluster.go

```diff
@@ -223,7 +221,7 @@ func (r *RemovalSimulator) findPlaceFor(removedNode string, pods []*apiv1.Pod, n
 
 	// remove pods from clusterSnapshot first
```

Comment:
should we change this comment to "// unscheduled pods from clusterSnapshot first" ?